check_and_gen.py

```diff
@@ -70,7 +70,7 @@
 
 tmprinter.out("Starting browser...")
 
-driverpath = get_driverpath()
+driverpath = "./chromedriver.exe"
```

Comment:
Using a hardcoded path is unacceptable

Reply:
Yes, we can't do this. :(

Reply:
This wasn't intended to be merged, did you two read the first sentence of the PR?

> I don't expect this to be merged, but figured I'd create it in case it helps anyone else save some time when trying to get GHunt working in the following: